'npm run build-server' doesn't work on Windows

``` 
npm run build-server

> vscode-xml@0.1.0 build-server C:\jbtgit\vscode-xml
> gulp build_server

[20:06:23] Using gulpfile C:\jbtgit\vscode-xml\gulpfile.js
[20:06:23] Starting 'build_server'...
'..' is not recognized as an internal or external command,
operable program or batch file.
[20:06:23] 'build_server' errored after 111 ms
[20:06:23] Error: Command failed: ../lsp4xml/mvnw.cmd clean verify -DskipTests
    at checkExecSyncError (child_process.js:481:13)
    at Object.execSync (child_process.js:521:13)
    at C:\jbtgit\vscode-xml\gulpfile.js:14:5
    at taskWrapper (C:\jbtgit\vscode-xml\node_modules\undertaker\lib\set-task.js                            :13:15)
    at bound (domain.js:280:14)
    at runBound (domain.js:293:12)
    at asyncRunner (C:\jbtgit\vscode-xml\node_modules\async-done\index.js:55:18)
    at _combinedTickCallback (internal/process/next_tick.js:73:7)
    at process._tickCallback (internal/process/next_tick.js:104:9)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vscode-xml@0.1.0 build-server: `gulp build_server`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vscode-xml@0.1.0 build-server script.
npm ERR! This is probably not a problem with npm. There is likely additional log                            ging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Snjeza\AppData\Roaming\npm-cache\_logs\2018-09-11T18_06_23                            _623Z-debug.log

```

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -11,7 +11,7 @@ gulp.task('tslint', () => {
 });
 
 gulp.task('build_server', function(done) {
-	cp.execSync(mvnw()+ " clean verify -DskipTests", {cwd:server_dir, stdio:[0,1,2]} );
+	cp.execSync(mvnw()+ " -o clean verify -DskipTests", {cwd:server_dir, stdio:[0,1,2]} );
   gulp.src(server_dir +'/org.eclipse.lsp4xml/target/org.eclipse.lsp4xml-all.jar')
 		.pipe(gulp.dest('./server'));
 	done();
@@ -31,5 +31,5 @@ function isLinux() {
 }
 
 function mvnw() {
-	return isWin()? server_dir+"/mvnw.cmd": server_dir+"/mvnw";
+	return isWin()? "mvnw.cmd": server_dir+"/mvnw";
 }

